{{ cookiecutter.package_name }}/setup.py

```diff
@@ -5,43 +5,53 @@
 import os
 import sys
 
+# Get some values from the setup.cfg
```

Comment:
I don't want to nit pick here too much, but this could be also added based on what's the value of the minimum version, I mean we may not need the try except for newer pythons